configs/config.json.example

```diff
@@ -382,16 +450,10 @@
         "Any pokemon put here directly force to use Berry & Best Ball to capture, to secure the capture rate": {},
         "any": {"catch_above_cp": 1200, "catch_above_iv": 0.9, "logic": "or" },
         "Lapras": {},
-        "Moltres": {},

```

Comment:
It doesn't hurt anything to place these inside an example file. So we should better keep it as it.